src/main/groovy/edu/oregonstate/mist/locations/MergeUtil.groovy

```diff
@@ -162,5 +163,17 @@ class MergeUtil {
             }
         }
 
+        def missingLocations = []
+        campusMapData.each { key, value ->
```

Comment:
I think you might be able to shorten this to something like this.
`def missingLocations = campusMapData.keySet() - data.collect { it.id }`